src/Providers/AppServiceProvider.php

```diff
@@ -84,6 +84,9 @@ public function boot(): void
         $this->setTimezone();
         $this->setNamespace();
         $this->bootAliases();
+        $this->loadViewsFrom("{$this->root}/resources/views", 'c');
+
+        Vite::useHotFile("{$this->root}/resources/hot");
```

Comment:
We already have this one as well https://github.com/craftcms/cms/blob/6.x/src/Twig/TwigServiceProvider.php#L15

I'd probably add a `ViewServiceProvider` and merge some of the things into that.